host/4/bullseye/amd64/java/java11/java11.Dockerfile

```diff
@@ -1,8 +1,13 @@
 # Build the runtime from source
 ARG HOST_VERSION=4.0.1.16815
-ARG JAVA_VERSION=11u11
+ARG JAVA_VERSION=11.0.12.7.1
+ARG BASE_JDK_URL=https://aka.ms/download-jdk
+ARG JAVA_HOME=/usr/lib/jvm/msft-11-x64
 FROM mcr.microsoft.com/dotnet/sdk:6.0.100-rc.2 AS runtime-image
```

Comment:
Is there any reason to use rc2?

Reply:
This has been removed in more recent versions of DEV.  you might have merge conflicts. But resolving those will be good.